Makefile

```diff
@@ -20,6 +20,7 @@ unexport SSH_CLIENT
 unexport SSH2_CLIENT
 
 NAME ?= ambassador
+DEV_KUBECONFIG ?= ~/.kube/config
```

Comment:
same feedback, this should probaly just fail and tell the user to export a DEV_KUBECONFIG

Reply:
I see the `preflight-dev-kubeconfig` target, which is great! need to drop this assignment too. 😉

Reply:
Done